packages/utils/upp.ts

```diff
@@ -3,6 +3,7 @@ import { TabDefinition } from "@/utils/types/tabs";
 
 export enum UppTabKeys {
   posts = "",
+  Subscrib = "premium-content",
```

Comment:
`Subcrib` with a `e` please. `Subscribe`, but i think `premiumSubscribe` is better for name it